feature-sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Sessions.kt

```diff
@@ -302,6 +302,12 @@ fun SessionsTopBar(
         (scheduleState as? Loaded)?.schedule?.days?.let { days ->
             TabRow(
                 selectedTabIndex = pagerState.currentPage,
+                modifier = Modifier
```

Comment:
Lovely!

I think it would be better to modify `pagerTabIndicatorOffset` and `Tab`'s padding because there is also a margin between the tabs. But your pr looks enough!